puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/wrappers/abort_build.yaml

```diff
@@ -0,0 +1,8 @@
+- wrapper:
+    name: abort_build
```

Comment:
Looks like those jobs have 60 and 30 minute timeouts -- do you want me to keep that as is or use 120 for all things?